src/linpred.jl

```diff
@@ -360,9 +360,21 @@ response(obj::LinPredModel) = obj.rr.y
 
 fitted(m::LinPredModel) = m.rr.mu
 predict(mm::LinPredModel) = fitted(mm)
-residuals(obj::LinPredModel) = residuals(obj.rr)
 
-function formula(obj::LinPredModel)
+function residuals(model::LinPredModel; type=:deviance)
```

Comment:
I do not know the details of the implementation, but is the `residuals` method properly documented (as it is not documented here). I.e. that it supports `type` kwarg and what is the behavior depending on the value of the kwarg.